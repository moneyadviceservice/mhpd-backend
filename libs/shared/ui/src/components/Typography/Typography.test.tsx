import React from 'react';
import { render, screen } from '@testing-library/react';
import { Typography } from '.';

describe('Typography component', () => {
  it('renders correctly', () => {
    render(<Typography />);
    const colors = screen.getByTestId('typography');
    expect(colors).toMatchSnapshot();
  });
});
