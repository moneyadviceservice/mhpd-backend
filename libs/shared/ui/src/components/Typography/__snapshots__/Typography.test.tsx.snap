// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Typography component renders correctly 1`] = `
<div
  data-testid="typography"
>
  <div
    class="mb-6"
  >
    <p
      class="text-xs"
    >
      the quick brown fox jumps over the lazy dog
    </p>
    <div>
      class: 
      <pre
        class="inline"
      >
        'text-
        xs
        '
      </pre>
    </div>
    <div>
      <p>
        font-size: 
        0.75rem
      </p>
      <p>
        line-height: 
        1.125rem
      </p>
    </div>
  </div>
  <div
    class="mb-6"
  >
    <p
      class="text-sm"
    >
      the quick brown fox jumps over the lazy dog
    </p>
    <div>
      class: 
      <pre
        class="inline"
      >
        'text-
        sm
        '
      </pre>
    </div>
    <div>
      <p>
        font-size: 
        1rem
      </p>
      <p>
        line-height: 
        1.625rem
      </p>
    </div>
  </div>
  <div
    class="mb-6"
  >
    <p
      class="text-base"
    >
      the quick brown fox jumps over the lazy dog
    </p>
    <div>
      class: 
      <pre
        class="inline"
      >
        'text-
        base
        '
      </pre>
    </div>
    <div>
      <p>
        font-size: 
        1.125rem
      </p>
      <p>
        line-height: 
        1.9375rem
      </p>
    </div>
  </div>
  <div
    class="mb-6"
  >
    <p
      class="text-lg"
    >
      the quick brown fox jumps over the lazy dog
    </p>
    <div>
      class: 
      <pre
        class="inline"
      >
        'text-
        lg
        '
      </pre>
    </div>
    <div>
      <p>
        font-size: 
        1.25rem
      </p>
      <p>
        line-height: 
        1.875rem
      </p>
    </div>
  </div>
  <div
    class="mb-6"
  >
    <p
      class="text-xl"
    >
      the quick brown fox jumps over the lazy dog
    </p>
    <div>
      class: 
      <pre
        class="inline"
      >
        'text-
        xl
        '
      </pre>
    </div>
    <div>
      <p>
        font-size: 
        1.375rem
      </p>
      <p>
        line-height: 
        2rem
      </p>
    </div>
  </div>
  <div
    class="mb-6"
  >
    <p
      class="text-2xl"
    >
      the quick brown fox jumps over the lazy dog
    </p>
    <div>
      class: 
      <pre
        class="inline"
      >
        'text-
        2xl
        '
      </pre>
    </div>
    <div>
      <p>
        font-size: 
        1.5rem
      </p>
      <p>
        line-height: 
        2.25rem
      </p>
    </div>
  </div>
  <div
    class="mb-6"
  >
    <p
      class="text-3xl"
    >
      the quick brown fox jumps over the lazy dog
    </p>
    <div>
      class: 
      <pre
        class="inline"
      >
        'text-
        3xl
        '
      </pre>
    </div>
    <div>
      <p>
        font-size: 
        1.75rem
      </p>
      <p>
        line-height: 
        2.625rem
      </p>
    </div>
  </div>
  <div
    class="mb-6"
  >
    <p
      class="text-4xl"
    >
      the quick brown fox jumps over the lazy dog
    </p>
    <div>
      class: 
      <pre
        class="inline"
      >
        'text-
        4xl
        '
      </pre>
    </div>
    <div>
      <p>
        font-size: 
        2.125rem
      </p>
      <p>
        line-height: 
        3.1875rem
      </p>
    </div>
  </div>
  <div
    class="mb-6"
  >
    <p
      class="text-5xl"
    >
      the quick brown fox jumps over the lazy dog
    </p>
    <div>
      class: 
      <pre
        class="inline"
      >
        'text-
        5xl
        '
      </pre>
    </div>
    <div>
      <p>
        font-size: 
        2.75rem
      </p>
      <p>
        line-height: 
        4.125rem
      </p>
    </div>
  </div>
  <div
    class="mb-6"
  >
    <p
      class="text-6xl"
    >
      the quick brown fox jumps over the lazy dog
    </p>
    <div>
      class: 
      <pre
        class="inline"
      >
        'text-
        6xl
        '
      </pre>
    </div>
    <div>
      <p>
        font-size: 
        3.5rem
      </p>
      <p>
        line-height: 
        4.375rem
      </p>
    </div>
  </div>
  <div
    class="mb-6"
  >
    <p
      class="text-7xl"
    >
      the quick brown fox jumps over the lazy dog
    </p>
    <div>
      class: 
      <pre
        class="inline"
      >
        'text-
        7xl
        '
      </pre>
    </div>
    <div>
      <p>
        font-size: 
        4rem
      </p>
    </div>
  </div>
  <div
    class="mb-6"
  >
    <p
      class="text-8xl"
    >
      the quick brown fox jumps over the lazy dog
    </p>
    <div>
      class: 
      <pre
        class="inline"
      >
        'text-
        8xl
        '
      </pre>
    </div>
    <div>
      <p>
        font-size: 
        5rem
      </p>
    </div>
  </div>
  <div
    class="mb-6"
  >
    <p
      class="text-9xl"
    >
      the quick brown fox jumps over the lazy dog
    </p>
    <div>
      class: 
      <pre
        class="inline"
      >
        'text-
        9xl
        '
      </pre>
    </div>
    <div>
      <p>
        font-size: 
        6rem
      </p>
    </div>
  </div>
</div>
`;
